v2/cmd/wails/init.go

```diff
@@ -230,6 +230,7 @@ func findAuthorDetails(options *templates.Options) {
 		name, err := git.Name()
 		if err == nil {
 			options.AuthorName = strings.TrimSpace(name)
+			options.AuthorName = strings.Replace(options.AuthorName, "\\", "", -1)
```

Comment:
This isn't removing an apostrophe from the name but rather a backslash.  Perhaps a function sanitizeName would be appropriate if there are additional characters to be removed in the future.  

Is this the only bad character that can occur?

Reply:
Sorry didn't get this, what is the change we are looking for ?

From what I understand, 
1. Create a function called sanitizeName
2. Remove apostrophe from the author name instead of back slash.

ANything else?

Reply:
My point is the change doesn't accomplish what your PR title says it does.

I am not sure what the correct output is based on this change set and your comments.